src/main/java/com/zendesk/maxwell/schema/Table.java

```diff
@@ -110,7 +110,7 @@ public Table copy() {
 		ArrayList<ColumnDef> list = new ArrayList<>();
 
 		for ( ColumnDef c : columnList ) {
-			list.add(c.copy());
```

Comment:
isn't this going to cause issue with alters? (that's my understanding of where those `copy` methods are used)